app/models/accession.rb

```diff
@@ -3,4 +3,6 @@
 # Model representing a single SDR item that is accessioned as part of a job run.
 class Accession < ApplicationRecord
   belongs_to :job_run
+
+  enum :state, in_progress: 'in_progress', completed: 'completed', failed: 'failed'
```

Comment:
cool, didn't know you could declare an `enum` like this!

Reply:
Yes, I greatly dislike numerical enums because they make it more difficult to work directly with the DB.

Reply:
for sure, same